src/util/duration.h

```diff
@@ -136,19 +157,19 @@ class Duration {
     }
 
     QString formatNanosWithUnit() const {
-        return QString("%1ns").arg(toNanos());
+        return QString("%1ns").arg(toIntegerNanos());
```

Comment:
I have just looked in the current log output and sorry to say, it is a regression. 

Debug [AnalyserQueue 1]: Waveform generation for track 9583 done 4.362s

looks really odd compared to a 

Debug [AnalyserQueue 1]: Waveform generation for track 9583 done 4.362 s

there is not a single line where the extra space causes a line wrap. 
And it is sufficiently documented, the a SI unit with no space before is just wrong.